content/tutorials/reactor-event-google.textile

```diff
@@ -34,19 +34,19 @@ meta_keywords: "Ably realtime, pizza ordering app, Google Cloud Functions, integ
 
-Even though Ably provides a full suite of "authentication and authorization features":https://ably.com/documentation/core-features/authentication to make sure users can only do what they're permitted to do, we still need a way to connect your messaging workflow to your business logic - something that would normally happen on your application's server. Doing so from a client-side browser application is risky, as it means exposing your business logic and other potentially sensitive data and services to the user. 
+Even though Ably provides a full suite of "authentication and authorization features":https://ably.com/documentation/core-features/authentication to make sure users can only do what they're permitted to do, we still need a way to connect your messaging workflow to your business logic - something that would normally happen on your application's server. Doing so from a client-side browser application is risky, as it means exposing your business logic and other potentially sensitive data and services to the user.
 
-With the rise of serverless solutions such as "Google Cloud Functions":https://cloud.google.com/functions/, the problem goes away. Create server functions that contain your business logic, credentials and other sensitive data, hook them up to your application, and all of the sensitive work is hidden away from the user, not to mention being managed by world-class infrastructure providers such as Amazon, Google or Microsoft. 
+With the rise of serverless solutions such as "Google Cloud Functions":https://cloud.google.com/functions/, the problem goes away. Create server functions that contain your business logic, credentials and other sensitive data, hook them up to your application, and all of the sensitive work is hidden away from the user, not to mention being managed by world-class infrastructure providers such as Amazon, Google or Microsoft.
 
-Ably's *"Reactor Events":https://ably.com/integrations* feature provides a way to define custom criteria that will automatically trigger different kinds of actions in response to channel messages, "channel lifecycle events":/realtime/channel-metadata#lifecycle-events (such as a channel being created) and presence events (such as users entering or leaving a channel), and is how we'll integrate with your server functions without exposing them to your users.
+Ably's *"Webhooks":https://ably.com/integrations* feature provides a way to define custom criteria that will automatically trigger different kinds of actions in response to channel messages, "channel lifecycle events":/realtime/channel-metadata#lifecycle-events (such as a channel being created) and presence events (such as users entering or leaving a channel), and is how we'll integrate with your server functions without exposing them to your users.
```

Comment:
> `*"Webhooks":https://ably.com/integrations*`

I'm not sure, but was there a reason this link was bolded? If not let's just make it a link.